lex/lexer.ml

```diff
@@ -146,7 +144,8 @@ let lex_ident lexbuf =
   end else raise Bad_rule
 
 let decimal_code  c d u =
-  100 * (Char.code c - 48) + 10 * (Char.code d - 48) + (Char.code u - 48)
+  100 * (Char.code c - Char.code '0') + 10 * (Char.code d - Char.code '0')
```

Comment:
small nitpick: but maybe breaking also before the first addition would make the code look more regular

Reply:
Sure, also fixed the commit message (Car -> Char)